### PR TITLE
Update Facebook

### PR DESCRIPTION
**Pull request by adamrusted**
facebook

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

File: icons/facebook.svg [hunk 1]
```diff
@@ -1 +1 @@
-<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Facebook</title><path d="M24 12.073c0-6.627-5.373-12-12-12s-12 5.373-12 12c0 5.99 4.388 10.954 10.125 11.854v-8.385H7.078v-3.47h3.047V9.43c0-3.007 1.792-4.669 4.533-4.669 1.312 0 2.686.235 2.686.235v2.953H15.83c-1.491 0-1.956.925-1.956 1.874v2.25h3.328l-.532 3.47h-2.796v8.385C19.612 23.027 24 18.062 24 12.073z"/></svg>
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Facebook</title><path d="M9.101 23.691v-7.98H6.627v-3.667h2.474v-1.58c0-4.085 1.848-5.978 5.858-5.978.401 0 .955.042 1.468.103a8.68 8.68 0 0 1 1.141.195v3.325a8.623 8.623 0 0 0-.653-.036 26.805 26.805 0 0 0-.733-.009c-.707 0-1.259.096-1.675.309a1.686 1.686 0 0 0-.679.622c-.258.42-.374.995-.374 1.752v1.297h3.919l-.386 2.103-.287 1.564h-3.246v8.245C19.396 23.238 24 18.179 24 12.044c0-6.627-5.373-12-12-12s-12 5.373-12 12c0 5.628 3.874 10.35 9.101 11.647Z"/></svg>
```
Comment (PratyushJayachandran) [inline icons/facebook.svg:1]: I see some [minor differences](https://github.com/simple-icons/simple-icons/assets/35000807/45a768f5-4c48-433c-9f9e-db62b6d024df) when comparing with the zip file in the source page.

**Review by PratyushJayachandran — CHANGES_REQUESTED** [review]

File: icons/facebook.svg (hunk 1 above)
Comment (adamrusted) [reply icons/facebook.svg:1]: I'm not seeing those differences. Did you subtract the white path from the blue before comparing?

**Review by PratyushJayachandran — APPROVED** [review]
Thank you for the contribution @adamrusted 🎉

File: icons/facebook.svg (hunk 1 above)
Comment (PratyushJayachandran) [reply icons/facebook.svg:1]: Didn't have to subtract, their brand kit had it already separated in the file. I compared again. Seems to be fine.

**Pull request merged**